sdk/storage/azblob/appendblob/models.go

```diff
@@ -164,3 +164,29 @@ func (o *SealOptions) format() (*generated.LeaseAccessConditions,
 }
 
 // ---------------------------------------------------------------------------------------------------------------------
+
+// ExpiryOptions defines values for ExpiryOptions
+type ExpiryOptions = generated.ExpiryOptions
```

Comment:
Do we actually need to export this?  It was replaced by `ExpiryType` yes?

Reply:
`ExpiryOptions` is used by the following method. We can remove it if we want.
```
// PossibleExpiryOptionsValues returns the possible values for the ExpiryOptions const type.
func PossibleExpiryOptionsValues() []ExpiryOptions {
	return generated.PossibleExpiryOptionsValues()
}
```

Reply:
Yes looks like this is remnants that can be removed.

Reply:
Done